src/main/resources/static/js/add-instances.js

```diff
@@ -2,6 +2,8 @@ jQuery(document).ready(function() {
 var frontendServiceUrl = $('#frontendServiceUrl').text();
 var frontendServicePath = "/backend";
 function instanceModel() {
+    var router = new Navigo(null, true, '#');
```

Comment:
Can you add a comment line here about what Navigo do?

Reply:
The router is initialized in a few places though, add-instances, switch-instances, login, and main. Should I comment in each file or perhaps only in main where the routes are defined?

Reply:
I chose to put the comment in main.js where the routes are first defined.

Reply:
Ok, I think we should have some comment about it, so users that have not worked with the code earlier can easier understand why Navigo is used.